### PR TITLE
Support list nesting

### PR DESCRIPTION
**Pull request by anurag90x**

### CODE REVIEW

File: markup_language/schema.xsd [hunk 1]
```diff
@@ -83,6 +83,7 @@
   <xs:complexType name="textContainer" mixed="true">
     <xs:choice minOccurs="0" maxOccurs="unbounded">
       <xs:group ref="formattedTextGroup"/>
+      <xs:group ref="listGroup"/>
```
Comment (markstory) [inline markup_language/schema.xsd]: Should you be able to put `<ul>` inside `<p>` tags? It looks like textContainer is mapped to `p` on line 110.

File: tests/documents/lists.html [hunk 2]
```diff
@@ -25,3 +25,34 @@
...
+          <li>
+            <em>"Nest more"</em>
+          </li>
+        </ul>
+      </li>
+    </ul>
+  </li>
+  <li>"Link text here"
+    <a href="https://wwww.google.com"/>
+    <img src="/service/uploads/images/test">
```
Comment (markstory) [inline tests/documents/lists.html]: Do we need to support `<p>` inside `li` elements?

Comment (anurag90x) [inline tests/documents/lists.html]: Froala doesnt seem to generate that so I would hold off on it for now.

Comment (markstory) [inline tests/documents/lists.html]: Ok.

File: markup_language/schema.xsd (hunk 1 above)
Comment (anurag90x) [inline markup_language/schema.xsd]: You can put lists inside headline tags and froala does generate that. Putting inside p tags is apparently not a thing as in browsers seem to just extract the list out and render it outside. Given that, I'm less inclined to bother about someone intentionally putting the list in the paragraph tag.

**Pull request merged**